conf/messages.cy

```diff
@@ -1500,3 +1498,21 @@ error.address.giveInLine1And2=Rhowch gyfeiriad yn llinell 1 a 2
 page.iht.application.assets.propertyType.nonResidential.label=Tir, adeilad dibreswyl neu fusnes
 page.iht.application.tnrbEligibilty.partner.additional.label.the.deceased={0}
 page.iht.registration.checklist.startRegistrationButton=Dechrau''r broses gofrestru
+page.iht.application.TnrbEligibilty.spouseOrCivilPartner.commonText=priod neu bartner sifil
+page.iht.application.tnrbEligibilty.partner.marriageOrCivilPartnership.label=briodas neu bartneriaeth sifil
+page.iht.application.tnrbEligibilty.partner.marriedOrCivilPartnership.label=yn briod neu mewn partneriaeth sifil
+page.iht.application.TnrbEligibilty.spouse.commonText=priod
+
+iht.estateReport.tnrb.partner.married=A fu {0} {1} ar unrhyw adeg tra''r oedd yn fyw, a bu farw''r {2}?
+iht.estateReport.tnrb.partner.hint=Os bu {0} {1} fwy nag unwaith tra''r oedd yn fyw, a bu farw''r {2} yn gyntaf bob tro, mae''n rhaid i chi ddewis un ystâd i drosglwyddo''r lwfans sydd heb ei ddefnyddio ohoni.
+
+page.iht.application.TnrbEligibilty.spouseOrCivilPartner.ofPerson = {1} {0}
+page.iht.application.tnrbEligibilty.overview.partner.name.question = {0}
+
+page.iht.application.tnrbEligibilty.partner.married.label = yn briod
+page.iht.application.tnrbEligibilty.partner.marriage.label = priodas
+page.iht.application.TnrbEligibilty.spouseOrCivilPartner.notOfPerson = {0} {1}
+
+page.iht.application.tnrbEligibilty.partner.additional.label.the.deceased.previous = previous?? {0}
+
+page.iht.application.tnrbEligibilty.partner.additional.label.their = their????
```

Comment:
It's merely a placeholder - we need the welsh and I've created another story to integrate this and some other content. The welsh is switched off at the moment and won't be put back on until the other story is completed.